src/Commands/Os2formsFailedJobsCommands.php

```diff
@@ -36,4 +36,14 @@ public function import(): void {
     $this->helper->handleImport();
   }
 
+  /**
+   * Remove entries from os2forms_failed_jobs_queue_submission_relation.
+   *
+   * @command os2forms_failed_jobs:remove-detached-relations
+   */
+  public function removeDetachedRelations(): void {
+    $relations = $this->helper->getDetachedQueueSubmissionRelations();
+    $this->helper->removeRelations($relations);
```

Comment:
```suggestion
    $this->helper->cleanUp();
```

cf. suggestion to create a `cleanUp` function.